src/main/java/uk/gov/hmcts/dts/fact/services/CourtService.java

```diff
@@ -88,17 +117,32 @@ public List<Court> getCourtsByCourtTypes(final List<String> courtTypes) {
             .collect(toList());
     }
 
+    /**
+     * Get courts by name or address or postcode or town fuzzy match.
+     *
+     * @return the list of courts
```

Comment:
I think the query param may have been missed here

Reply:
added missing bit